examples/nD_labels.py

```diff
@@ -10,6 +10,7 @@
 
 with napari.gui_qt():
     blobs = data.binary_blobs(length=128, volume_fraction=0.1, n_dim=3)
-    viewer = napari.view(blobs=blobs.astype(float))
+    viewer = napari.Viewer()
+    viewer.add_image(blobs.astype(float))
```

Comment:
Why change this?

Reply:
wasn't intentional, was playing around with the file during development, and didn't revert. i'm reverting now